jobs/starter/upgrade/Jenkinsfile

```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env groovy
 
-def mail_success(list,detail) {
+def mail_success(list,detail,warn) {
+    body = "Cluster ${CLUSTER_NAME} upgrade details:\n"
+    if ( warn ) {
+        body += "\nWARNING: post-upgrade smoke test was not successful!\n"
+        body += "WARNING: smoke test job details: ${warn}\n"
```

Comment:
Redundant "WARNING" ?

Reply:
This was intentional, for visual purposes... I thotught it makes it more obvious and easier to read/spot the smoke test URL vs the upgrade job URL which I also moved to the top for [hopefully] easier access.

A notification with smoke test warnings would look like this:

```
Cluster CLUSTER_NAME upgrade details:

WARNING: post-upgrade smoke test was not successful!
WARNING: smoke test job details: https://jenkins-jenkins.192.168.122.78.nip.io/job/smoketest/25/

Upgrade Jenkins job: https://jenkins-jenkins.192.168.122.78.nip.io/job/test/7/

<cluster status>

Smoke test results: FAILURE
```

I can leave the upgrade job URL at the bottom and supress the second WARNING line (leaving the smoke test URL only) if preferred

Reply:
Maybe combining the lines is sufficient. I just don't want it to look as scary :)
WARNING: Post-upgrade smoke test was not successful: URL